gradle.properties

```diff
@@ -2,3 +2,5 @@ org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.workers.max=6
 org.gradle.jvmargs=-Xmx3g
+org.gradle.java.installations.paths=/Users/xchen21/Documents/JDK/8/zulu-8.jdk,/Users/xchen21/Documents/JDK/11/zulu-11.jdk,/Users/xchen21/Documents/JDK/17/zulu-17.jdk,/Users/xchen21/Documents/JDK/21/zulu-21.jdk
```

Comment:
Let's revert this change :)

Reply:
@SiBorea you can add this line to your personal ~/.gradle/gradle.properties file to have it in effect for all your gradle builds without having to edit any of them directly